python/scripts/build_python_package.py

```diff
@@ -151,5 +156,35 @@ def patch_line_matching_prefix(file_path: Path, prefix: str, new_line: str) -> N
     file_path.write_text("\n".join(lines))
 
 
+def check_readme_links(readme_path: Path) -> None:
+    """Check if the README contains valid absolute links."""
```

Comment:
Improve docs: mention that the function makes the script exit with error code in case of problems.